patroni/ctl.py

```diff
@@ -1104,20 +1106,22 @@ def restart(cluster_name: str, group: Optional[int], member_names: List[str],
                                  type=str, default='now')
 
     scheduled_at = parse_scheduled(scheduled)
-    confirm_members_action(members, force, 'restart', scheduled_at)
+
+    content: Dict[str, Any] = {}
+    if pending:
+        content['restart_pending'] = True
+        members = [m for m in members if m.data.get('pending_restart', False)]
```

Comment:
How would that look like?  If no restart is pending, do nothing?  And then say something in the confirmation message like 'In case there will be changes requiring a restart, it will be done on those nodes', without mentioning any node?

Also, this means reevaluating the member list at the scheduled time.  I have to do my research to figure out how to do that.